### PR TITLE
Create akhilesh-w.json

### PR DESCRIPTION
**Pull request by akhilesh-w**

### CODE REVIEW

File: data/akhilesh-w.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "Akhilesh Waghmare",
+  "type": "personal",
+  "bio": "Designer & Developer"
```
Comment (imbeshat) [inline data/akhilesh-w.json:4]:
```suggestion
  "bio": "Designer & Developer",
```

File: data/akhilesh-w.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/the_akhilesh_w",
+      "icon": "FaTwitter"
+    }
```
Comment (imbeshat) [inline data/akhilesh-w.json:10]:
```suggestion
    },
```

File: data/akhilesh-w.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+   {
+      "name": "Follow me on Instagram",
+      "url": "https://twitter.com/dichotomyofakhil",
+      "icon": "FaInstagram"
+    }
```
Comment (imbeshat) [inline data/akhilesh-w.json:15]:
```suggestion
    },
```

File: data/akhilesh-w.json [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+   {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/akhilesh-waghmare-0652a31b9/",
+      "icon": "FaLinkedIn"
+    }
```
Comment (imbeshat) [inline data/akhilesh-w.json:20]:
```suggestion
    },
```

File: data/akhilesh-w.json [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+  "bio": "Designer & Developer"
+    "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/the_akhilesh_w",
+      "icon": "FaTwitter"
+    }
+   {
+      "name": "Follow me on Instagram",
+      "url": "https://twitter.com/dichotomyofakhil",
```
Comment (imbeshat) [inline data/akhilesh-w.json:13]: should be Instagram URL.

**Review by imbeshat — CHANGES_REQUESTED** [review]

File: data/akhilesh-w.json (hunk 3 above)
Comment (Amishapopli1) [reply data/akhilesh-w.json]: "url": "https://instagram.com/dichotomyofakhil".

File: data/akhilesh-w.json
Comment (Amishapopli1) [inline data/akhilesh-w.json:1]: use instagram url

**Comment by akhilesh-w** [comment]
Thanks for the link review @imbeshat

File: data/akhilesh-w.json [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+    },
+   {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/akhilesh-waghmare-0652a31b9/",
+      "icon": "FaLinkedIn"
+    },
+   {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/akhilesh-w",
+      "icon": "FaGitHub"
```
Comment (khushalarora) [inline data/akhilesh-w.json:24]: Replace it with **' FaGithub '** :+1: 
```Suggestion
      "icon": "FaGithub"
```

**Review by khushalarora — CHANGES_REQUESTED** [review]
Hi @akhilesh-w, thank you for contributing :tada:

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**